gunicorn.conf.py

```diff
@@ -0,0 +1,69 @@
+import uvicorn
+
+
+# Debugging
+reload = False
+reload_engine = 'auto'
+spew = False
+check_config = False
+print_config = False
+
+
+# Server socket
+bind = '127.0.0.1:8000'
```

Comment:
You can setup bind dynamically through the settings managements (HOST, PORT) parameters.

Reply:
Did it